src/content/docs/es/guides/cms/payload.mdx

```diff
@@ -133,7 +133,7 @@ Por ejemplo, para mostrar una lista de títulos de publicaciones y su contenido:
 ---
 import HomeLayout from "../layouts/HomeLayout.astro";
 
-const res = await fetch("https://localhost:5000/api/posts") // https://localhost:3000/api/posts by default
+const res = await fetch("https://localhost:5000/api/posts") // https://localhost:4321/api/posts by default
```

Comment:
I guess we should reflect `http` instead of `https` here

```suggestion
const res = await fetch("http://localhost:5000/api/posts") // http://localhost:4321/api/posts by default
```